test/flipper_app/authorize_test.rb

```diff
@@ -0,0 +1,75 @@
+require_relative '../test_helper'
+require 'mocha/setup'
+require 'flipper_app/authorize'
+
+class AssignmentsApiTest < Minitest::Test
+  def test_unauthenticated_users_rejected
+    User.stubs(:where).returns(authorized_users)
+    env = {'rack.session' => {}}
+    response = middleware_response(env)
+    assert_forbidden(response)
+  end
+
+  def test_nil_session_rejected
+    User.stubs(:where).returns(authorized_users)
+    env = {'rack.session' => nil}
+    response = middleware_response(env)
+    assert_forbidden(response)
+  end
+
+  def test_unauthorized_users_rejected
+    User.stubs(:where).returns(authorized_users)
+    env = {'rack.session' => {'github_id' => normal_user_github_id}}
+    response = middleware_response(env)
+    assert_forbidden(response)
+  end
+
+  def test_authorized_user_allowed
+    User.stubs(:where).returns(authorized_users)
+    env = {'rack.session' => {'github_id' => admin_github_id}}
+    response = middleware_response(env)
+    assert_allowed(response)
+  end
+
+  def test_development_env_allowed
+    User.stubs(:where).returns(authorized_users)
+    env = {'rack.session' => nil}
+    response = force_development_env { middleware_response(env) }
+    assert_allowed(response)
```

Comment:
Maybe  `test_flipper_management_allowed_in_development` would be more expressive here?